addons/sms/models/mail_message.py

```diff
@@ -34,13 +34,13 @@ def _search_has_sms_error(self, operator, operand):
             return ['&', ('notification_ids.notification_status', '=', 'exception'), ('notification_ids.notification_type', '=', 'sms')]
         raise NotImplementedError()
 
-    def message_format(self):
+    def message_format(self, fetch_followers=False):
         """ Override in order to retrieves data about SMS (recipient name and
             SMS status)
 
         TDE FIXME: clean the overall message_format thingy
         """
-        message_values = super(MailMessage, self).message_format()
+        message_values = super(MailMessage, self).message_format(fetch_followers)
```

Comment:
Use keyword instead of position to make the override a bit more robust to changes.